bcgov/HOUS-permit-portal
# Keithmaibach/sc 67666/manage local jurisdictions jurisdiction index

## Description

add admin/manger home screens
the index UI layout, unwired but styled paginator, model methods, blueprints, and importer script for jurisdictions. Fixes specs and seeders.

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature
- [ x] 📦 Chore (Release)

## Related Tickets & Documents

https://app.shortcut.com/codechallengell/story/67666/manage-local-jurisdictions-jurisdiction-index-super-admin
https://app.shortcut.com/codechallengell/story/67452/please-provide-a-list-of-local-jurisdictions-for-the-province-perferablling-in-a-csv-or-excel-format
https://app.shortcut.com/codechallengell/story/67548/home-page-super-admin

## Steps to QA

Reset DB and seed
Check out http://localhost:3000/jurisdictions
real data! but no search, sort, or pagination...

## UI Accessibility Checklist

Wave checker good

_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

- [ x] Markup uses semantic HTML?
- [ x] Additional context added through `aria-roles` and `aria-labels`?
- [x ] Checked
      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?
- [x ] Usable with a keyboard and navigable in a logical order?
- [ x] Usable using a screen reader (e.g. Voiceover for Mac) with enough context for the user?

## General Checklist

- [ x] ✅ Provide tests for your changes where relevant
- [x ] 📝 Use descriptive commit messages
- [ x] 📗 Update any related documentation and include any relevant screenshots
- [x ] 📱 For UI-related tasks, ensure responsive design is implemented across multiple screen size
- [ x] 🗂️ Move your relevant story/task to the _Ready for Code Review_ state
